app/views/pages/home.html.erb

```diff
@@ -1,6 +1,51 @@
+<div class="container-1200">
 
-<%# <h1>Pages#home</h1>
-<p>Find me in app/views/pages/home.html.erb</p>
-<p>Hello its antonio</p>
-<p>Hello, it's Mariana</p>
-<p>Hi, it's Louis</p> %>
+  <div class="banner">
+    <div class="titles">
+
+      <h3>Welcome to</h3>
+      <h1>Rentart.</h1>
+      <h3>Do you want to rent the Mona Lisa for a day?</h3>
+
+      <div class="btn-index">
+        <%= link_to "Book a piece of art", root_path, class: "btn-gold" %>
+      </div>
+
+    </div>
+    <div>
+    <%= image_tag "home_page_painting", class: "image" %>
+    </div>
+  </div>
+
+  <div class="sign-in">
+    <h2>Not registered yet?</h2>
+    <p>render form</p>
+    <%= image_tag "sign_up", class: "sign-up-img" %>
+  </div>
+
+  <div class="categories">
+
+    <h2>Find by category</h2>
+
+    <div class="category-cards">
+      <div class="card-category">
+        <%= image_tag "painting_category", class: "category-image" %>
+        <div class="card-infos">
+          <h3><%= link_to "Paintings", root_path, class: "text-category-card" %></h3>
+        </div>
+      </div>
+      <div class="card-category">
+        <%= image_tag "sculpture_category", class: "category-image" %>
+        <div class="card-infos">
+          <h3><%= link_to "Sculptures", root_path, class: "text-category-card" %></h3>
+        </div>
+      </div>
+      <div class="card-category">
+        <%= image_tag "instalation_category", class: "category-image" %>
+        <div class="card-infos">
+          <h3><%= link_to "Instalations", root_path, class: "text-category-card" %></h3>
+        </div>
+      </div>
+    </div>
+  </div>
+</div>
```

Comment:
Usually you add white spaces between siblings, not between children and parents.

For example
```
div
  h1
  div
    h2
    p
  /div
  div
    p
     span
    /p
  /div
/div    
```

would be 
```
div
  h1
  
  div
    h2
    p
  /div
  
  div
    p
     span
    /p
  /div
/div    
```

Can you try something similar to that ?